### DISCUSSION TITLE
Can I make tab invisible?

### DISCUSSION THREAD

**Discussion by Demianeen** [LazyVim/LazyVim#2932] [Q&A] [answered]
I recently decided to use tabs instead of spaces by default in nvim and was a bit confused when I started to see tabs:

![CleanShot 2024-04-05 at 12 29 06@2x](https://github.com/LazyVim/LazyVim/assets/51330172/e0f2225a-deab-4c82-adf3-4d09fc71b25c)

Can I make them invisible, similar to space?

**Comment by dpetka2001** [comment, marked as answer]
You can either add `vim.opt.list = false` in your `options.lua` file (but that will disable or `listchars`) or you can specify `vim.opt.listchars` how you want the chars to look like. For more info see `:h 'listchars'`.
